### ISSUE TITLE
Object.addRightTile is deprecated.

### ISSUE THREAD

**Issue by andya9** [nesQuick/atom-wordcount#30]
Use version ^1.0.0 of the status-bar Service API.
```
Object.addRightTile (/usr/share/atom/resources/app/node_modules/status-bar/lib/main.js:178:16)
Object.consumeStatusBar (/data/linux/configs/atom/packages/wordcount/lib/main.coffee:36:21)
```

**Comment by nesQuick** [comment]
I'm happy to take your pull requests to get this fixed :)

**Comment by nesQuick** [comment]
fixed with 2.2.2
